src/components/checkbox/checkbox.scss

```diff
@@ -17,6 +17,7 @@
 }
 
 :host(limel-checkbox) {
+    min-height: 2.5rem; // ensures that the UI doesn't flicker too much when switching to `readonly`
```

Comment:
seems to cause this bug
<img width="202" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/37984119-2d6e-431e-897d-a1f42e13e741">